Add docs for Libre Hardware Monitor integration

Sab44 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Added docs for new Libre Hardware Monitor integration.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/140449
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/6731
- This PR fixes or closes issue: n/a

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced a new integration enabling system monitoring with Libre Hardware Monitor in Home Assistant.

- **Documentation**
  - Added comprehensive setup instructions covering prerequisites, configuration options, reconfiguration procedures, troubleshooting tips, and removal guidelines.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Sab44 committed:
add docs for librehardwaremonitor

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+ha_quality_scale: bronze
+---
+
+The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors that will display system information.
```
```suggestion
The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors and display that system information in Home Assistant.

Libre Hardware Monitor is an [Open Source](https://www.majorgeeks.com/content/page/what_is_open_source.html) fork of [Open Hardware Monitor](https://www.majorgeeks.com/files/details/open_hardware_monitor.html) to monitor your computer's sensors such as temperature sensors, fan speeds, voltages, load, and clock speeds.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+To open port (on Windows):
+
+1. Navigate to Control Panel, System and Security and Windows Firewall.
```
```suggestion
1. In Windows, Navigate to Control Panel, System and Security and Windows Firewall.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+## Prerequisites
+
+Libre Hardware Monitor must be running on the host, with "Remote web server" active. You also need to open inbound port (TCP 8085) on the host.
```
```suggestion
- Libre Hardware Monitor is installed on the system (host) you want to monitor.
- Libre Hardware Monitor must be running.
- In Libre Hardware Monitor, make sure **Remote web server** active. 
- On the host system, open the inbound port TCP 8085.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+Libre Hardware Monitor must be running on the host, with "Remote web server" active. You also need to open inbound port (TCP 8085) on the host.
+
+To open port (on Windows):
```
```suggestion
### To open a port (on Windows):
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+1. Navigate to Control Panel, System and Security and Windows Firewall.
+2. Select **Advanced settings** and highlight **Inbound Rules** in the left pane.
+3. Right click Inbound Rules and select New Rule.
```
```suggestion
3. Right-click **Inbound Rules** and select **New Rule**.
```
We mark UI strings bold

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+1. Navigate to Control Panel, System and Security and Windows Firewall.
+2. Select **Advanced settings** and highlight **Inbound Rules** in the left pane.
+3. Right click Inbound Rules and select New Rule.
+4. Add the port you need to open and click Next.
```
```suggestion
4. Add the port you need to open and select **Next**.
```
In general, we avoid [click and use select instead](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+2. Select **Advanced settings** and highlight **Inbound Rules** in the left pane.
+3. Right click Inbound Rules and select New Rule.
+4. Add the port you need to open and click Next.
+5. Add the protocol (TCP) and the port number (8085) into the next window and click Next.
```
```suggestion
5. Add the protocol (TCP) and the port number (8085) into the next window and select **Next**.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+3. Right click Inbound Rules and select New Rule.
+4. Add the port you need to open and click Next.
+5. Add the protocol (TCP) and the port number (8085) into the next window and click Next.
+6. Select Allow the connection in the next window and hit Next.
```
```suggestion
6. In the next window, select **Allow the connection**, then select **Next**.
```
location before action

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+4. Add the port you need to open and click Next.
+5. Add the protocol (TCP) and the port number (8085) into the next window and click Next.
+6. Select Allow the connection in the next window and hit Next.
+7. Select the network type as you see fit and click Next.
```
```suggestion
7. Select the network type as you see fit and select **Next**.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+5. Add the protocol (TCP) and the port number (8085) into the next window and click Next.
+6. Select Allow the connection in the next window and hit Next.
+7. Select the network type as you see fit and click Next.
+8. Name the rule and click Finish.
```
```suggestion
8. Name the rule and select **Finish**.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+{% configuration_basic %}
+host:
+  description: IP address or hostname of the system where Libre Hardware Monitor is running.
```
```suggestion
  description: IP address or hostname of the system where Libre Hardware Monitor is running. The system you want to monitor.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+## Reconfiguration
+
+This integration allows you to reconfigure an existing entry, e.g. if you want to add or remove hardware devices.
```
```suggestion
This integration allows you to reconfigure an existing entry. For example, if you want to add or remove hardware devices.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+This integration allows you to reconfigure an existing entry, e.g. if you want to add or remove hardware devices.
+
+1. Go to **{% my integrations title="Settings > Devices & services" %}**.
```
```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & services**" %}.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+1. Go to **{% my integrations title="Settings > Devices & services" %}**.
+2. Select the Libre Hardware Monitor integration.
+3. To reconfigure the settings, click {% icon "mdi:dots-vertical" %} to the right of an entry and select **Reconfigure**.
```
```suggestion
3. To reconfigure the settings, to the right of an entry, select the three dots  {% icon "mdi:dots-vertical" %} menu and select **Reconfigure**.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+## Known limitations
+
+Servers which require authentication are currently not supported.
```
what does this mean? If I have a computer with a password protected login I can't use Libre Hardware Monitor on it?

Sab44 replied:
I see how that could be confusing. In Libre Hardware Monitor you can set up additional authentication for the remote web server, so requiring username & password to access the web interface. I have rewritten that sentence to clarify this.
I plan on supporting authentication in the near future, but did not want to blow this PR up more than needed at the moment.

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+### Problem with connection during setup
+
+Open a browser and navigate to `http://<IP address>:<Port>` of your Libre Hardware Monitor instance, make sure you can see and update the data there.
```
```suggestion
Check if Libre Hardware Monitor is running properly. On the system running Libre Hardware Monitor, open a browser and navigate to `http://<IP address>:<Port>`. Make sure you can see and update the data there.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+### Sensors change to "unavailable" status
+
+This is expected behavior when the computer is not reachable, e.g., it is turned off. Sensors will resume their data readings once the computer is reachable again.
```
```suggestion
This is expected behavior when the system you are monitoring is not reachable. For example, because it is turned off. Sensors will resume their data readings once the system is reachable again.
```

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,78 @@
+
+### Integration stops working
+
+Make sure the IP address of your computer running Libre Hardware Monitor has not changed. Ideally, set a static IP address for that computer in your router.
```
```suggestion
Make sure the IP address of the system you are monitoring has not changed. Ideally, set a static IP address for that system in your router.
```

c0ffeeca7 reviewed (changes requested):
I took quite a few liberties rephrasing. Please correct me if I got something wrong.

Sab44 commented:
> I took quite a few liberties rephrasing. Please correct me if I got something wrong.

Great feedback, thanks. I have addressed all of your suggestions.

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,86 @@
+ha_quality_scale: bronze
+---
+
+The `librehardwaremonitor` {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors to display that system information in Home Assistant.
```
```suggestion
The **Libre Hardware Monitor** {% term integration %} uses your [Libre Hardware Monitor](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor) installation as a source for sensors to display that system information in Home Assistant.
```
we no longer use the code-fenced version to spell the integration name for integrations that have a config flow. you might still see it for those that are added via YAML config file.

c0ffeeca7 commented on source/_integrations/librehardwaremonitor.markdown:
```diff
@@ -0,0 +1,86 @@
+description: Instructions on how to integrate Libre Hardware Monitor within Home Assistant.
+ha_category:
+  - System monitor
+ha_release: 2025.4
```
```suggestion
ha_release: 2025.4
ha_config_flow: true
ha_codeowners:
  - '@Sab44'
```

c0ffeeca7 reviewed (dismissed):
Thank you, @Sab44 :+1:

Sab44 committed:
remove scan interval config option and reconfigure docs

Sab44 commented:
Updated to reflect changes done on the core PR.

Sab44 committed:
improve formatting

Sab44 committed:
Add underscores to integration name

Sab44 committed:
Add note about connection loss handling

c0ffeeca7 commented on source/_integrations/libre_hardware_monitor.markdown:
```diff
@@ -0,0 +1,81 @@
+description: Instructions on how to integrate Libre Hardware Monitor within Home Assistant.
+ha_category:
+  - System monitor
+ha_release: 2025.4
```
```suggestion
ha_release: 2025.8
```
tbd

joostlek replied:
```suggestion
ha_release: "2025.10"
```

c0ffeeca7 commented on source/_integrations/libre_hardware_monitor.markdown:
```diff
@@ -0,0 +1,81 @@
+- Libre Hardware Monitor must be running during setup.
+- In Libre Hardware Monitor, make sure **Remote web server** is active.
+- Make sure to open the inbound port (8085 by default) on the host system's firewall.
+- In Libre Hardware Monitor go to **File** → **Hardware** and check the devices you want to monitor.
```
```suggestion
- In Libre Hardware Monitor, go to **File** > **Hardware** and check the devices you want to monitor.
```

c0ffeeca7 commented on source/_integrations/libre_hardware_monitor.markdown:
```diff
@@ -0,0 +1,81 @@
+
+### To open a port (on Windows Firewall)
+
+1. In Windows, navigate to **Control Panel** → **System and Security** → **Windows Defender Firewall**.
```
```suggestion
1. In Windows, navigate to **Control Panel** > **System and Security** > **Windows Defender Firewall**.
```

c0ffeeca7 reviewed (commented):
tiny style tweaks

c0ffeeca7 committed:
Apply suggestions from code review

c0ffeeca7 reviewed (dismissed):
Thank you, @Sab44 👍

joostlek committed:
Update source/_integrations/libre_hardware_monitor.markdown
